packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -420,18 +446,24 @@ export const useArbTokenBridge = (
     return { estimatedL1Gas, estimatedL2Gas, estimatedL2SubmissionCost }
   }
 
+  // TODO: add RequestObject as a possible parameter?
+  // Or is RequestObject the same as current parameters and user will populate them as usual?
```

Comment:
Moving to `RequestObject` isn't the highest priority for this upgrade, we can leave it for a future one.